.github/workflows/release.yml

```diff
@@ -189,3 +189,93 @@ jobs:
           repository: observIQ/otel-collector-installer-testing
           event-type: upstream_prerelease
           client-payload: '{ "version": "${{ github.ref_name }}" }'
+
+      # Linux package uploads to Google Artifact Registry
+
+      # google-github-actions/auth@v2 was used above to install
+      # gcloud sdk. google-github-actions/auth was used as well
+      # but we need to re-authenticate to upload packages to the
+      # correct project.
+      - name: "Authenticate to Google Cloud"
+        uses: "google-github-actions/auth@v2"
+        with:
+          credentials_json: "${{ secrets.ORG_BINDPLANE_PUBLIC_LINUX_REPO_JSON_KEY }}"
+
+      - name: Upload Bindplane Linux AMD64 Deb Package
+        run: |
+          gcloud \
+            --project bindplane-repository \
+            artifacts apt upload stable-deb \
+            --location us-central1 \
+            --source dist/observiq-otel-collector_${{ github.ref_name }}_linux_amd64.deb
```

Comment:
`${{ github.ref_name }}` should evaluate to the pushed tag. These steps are at the end of the workflow, it is okay if they fail during a release as we do not rely on artifact registry at this time.